v20.1/alter-primary-key.md

```diff
@@ -0,0 +1,157 @@
+---
+title: ALTER PRIMARY KEY
+summary: Use the ALTER PRIMARY KEY statement to change the primary key of a table.
+toc: true
+---
+
+ <span class="version-tag">New in v20.1:</span> The `ALTER PRIMARY KEY` [statement](sql-statements.html) is a subcommand of [`ALTER TABLE`](alter-table.html) that can be used to change the [primary key](primary-key.html) of a table.
+
+{{site.data.alerts.callout_danger}}
+This is currently an experimental feature. The interface and output are subject to change.
+
+To enable online primary key changes, set the `experimental_enable_primary_key_changes` [session variable](experimental-features.html#session-variables) to `true`.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}
+When you change a primary key with `ALTER PRIMARY KEY`, the old primary key index becomes a secondary index. This helps optimize the performance of queries that still filter on the old primary key column, but it makes `ALTER PRIMARY KEY` an expensive operation, as it performs rewrites of multiple indexes.
```

Comment:
The rewrite of the old primary key into a secondary index isn't the reason why alter primary key is expensive. alter primary key is expensive just because it has to rewrite a new index + multiple indexes other than the old primary key being rewritten. Do we have a precedent of linking to rfc's/internal docs for discussion of more detailed information? If so, we could show the rules for which indexes in the table get rewritten.